network-api/networkapi/templates/buyersguide/pages/consumer_creepometer_page.html

```diff
@@ -0,0 +1,17 @@
+{% extends "pages/buyersguide/base.html" %}
```

Comment:
The two new template files are created under `network-api/networkapi/templates/buyersguide/pages` as I thought this pattern aligns better with our cleanup work (https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10492)?